README.md

```diff
@@ -307,6 +307,9 @@ from information_schema.procedures
 where COVERAGE < 75;
 ```
 
+### Logging
+SnowCLI print logs on console. On default, it will print only error level logs, if you need more information you can run command with parameter `-v/--verbose`, this will print logs with level info and higher. If it still not enough, there is `--debug` parameter which will print logs with level debug and higher with additional information.
```

Comment:
```suggestion
SnowCLI print logs to console. By default, it will print only error level logs, if you need more information you can run command with parameter `-v/--verbose`, this will print logs with level info and higher. If it still not enough, there is `--debug` parameter which will print logs with level debug and higher with additional information.
```

Can you also show how will error log look like in cli?

Reply:
changed